xbmc/cores/AudioEngine/AEDefines.h

```diff
@@ -22,5 +22,7 @@
 #define AE_AC3_ENCODE_BITRATE 640000
 #define AE_DTS_ENCODE_BITRATE 1411200
 
+#ifdef BUILDING_WITH_CMAKE
```

Comment:
It's a cmake define, like the one for SSE the one for X11, used e.g. here: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp#L49 the one for MIR, the one for whatever ... can you explain me the difference between your problem with the ifdef in this place and the ifdef that you pushed some weeks ago into VAAPI?

I think I have a problem calibrating towards your wishes.